idb/dummy.go

```diff
@@ -217,29 +235,34 @@ type IndexerDb interface {
 
 	LoadGenesis(genesis types.Genesis) (err error)
 	SetProto(version string, proto types.ConsensusParams) (err error)
-	GetProto(version string) (proto types.ConsensusParams, err error)
+	GetProtoTx(tx *sql.Tx, version string) (proto types.ConsensusParams, err error)
 
 	GetImportState() (is *ImportState, err error)
 	SetImportState(ImportState) (err error)
-	GetMaxRoundAccounted() (round uint64, err error)
+	GetMaxRoundAccountedTx(tx *sql.Tx) (round uint64, err error)
 	GetMaxRoundLoaded() (round uint64, err error)
-	GetSpecialAccounts() (SpecialAccounts, error)
+	GetSpecialAccountsTx(ctx context.Context, tx *sql.Tx) (SpecialAccounts, error)
```

Comment:
The context is for operations which utilize a channel for some potentially long operation, I don't think you need to add that here or to the Health endpoint.